x/auth/src/client/grpc.rs

```diff
@@ -41,6 +42,7 @@ where
         &self,
         request: Request<QueryAccountRequest>,
     ) -> Result<Response<QueryAccountResponse>, Status> {
+        info!("Received a gRPC request auth::account");
```

Comment:
I think that there should be any way to omit this. Maybe write some middleware for server